src/components/ChatContent/ChatContent.js

```diff
@@ -420,6 +448,9 @@ export default class ChatContent extends React.Component {
             dataPageSize={this.props.dataPageSize}
             isResizing={this.props.isResizing}
             shouldRender={this.props.shouldRender}
+            setAxiosSource={(axiosSource) => {
```

Comment:
Instead of using a callback to set another axios source, you can just call `queryInputRef.cancelQuery()` and it should handle everything for you